app/controllers/api/connect/v3/systems/systems_controller.rb

```diff
@@ -19,4 +21,9 @@ def deregister
     respond_with(@system.destroy, serializer: ::V3::SystemSerializer)
   end
 
+  private
+
+  def hwinfo_params
```

Comment:
Please `hw_info_params` instead of `hwinfo_params` if you already set the standard name in the database.